netshare/generate/generate.py

```diff
@@ -0,0 +1,26 @@
+from netshare.generate.denormalize_fields import denormalize_fields
+from netshare.generate.generate_api import get_generated_data_dir
+from netshare.generate.model_generate import model_generate
+from netshare.utils.paths import copy_files
+
+
+def generate() -> None:
+    """
+    This is the main function of the postprocess phase.
+    We get the generated data, prepare it to be exported, export it, and create the visualization.
+
+    This function execute the following steps:
+    1. Denormalize the fields (e.g. int to IP, vector to word, etc.)
+    2. Choose the best generated data and write it using the generate_api
+    """
+    model_generate()
+    denormalized_fields_dir = denormalize_fields()
+    chosen_data_dir = choose_best_chunk(denormalized_fields_dir)
+    copy_files(chosen_data_dir, get_generated_data_dir())
+
+
+def choose_best_chunk(denormalized_fields_dir: str) -> str:
```

Comment:
I personally think the function name should not be `choose_best_chunk` because for each chunk of data, we will generate several models with different hyper-parameters. This function is to pick the best model for each chunk.

Reply:
You're absolutely right, I'll rename to: `choose_best_model`